<?xml version="1.0" ?>
<testsuite errors="1" failures="1" name="fts.index_management_api.IndexManagementAPI" skip="0" tests="1" time="50.7702651024">
	<testcase name="fts.index_management_api.IndexManagementAPI.test_planfreeze_control,doc-per-day:1,GROUP:P1,index_type:scorch,spec:test,fts_quota:750,conf_file:test.conf,num_nodes:4,cluster_name:test,cluster:D+F+Q+I,stop-on-failure:False,ini:test.ini,case_number:1,get-logs:False,scan_consistency:NOT_BOUNDED,logs_folder:/Users/girishbenakappa/IdeaProjects/testrunner/logs/testrunner-19-Aug-15_15-43-44/test_1,bucket_size:200,disable_HTP:True" time="50.7702651024">
		<error type="membase.error">['Traceback (most recent call last):\n  File &quot;pytests/fts/index_management_api.py&quot;, line 11, in setUp\n    super(IndexManagementAPI, self).setUp()\n  File &quot;pytests/fts/fts_base.py&quot;, line 3105, in setUp\n    self.__setup_for_test()\n  File &quot;pytests/fts/fts_base.py&quot;, line 3241, in __setup_for_test\n    self._input.servers[1:])\n  File &quot;pytests/fts/fts_base.py&quot;, line 1881, in init_cluster\n    &quot;%s: %s&quot; % (cluster_services, e))\nFTSException: Unable to initialize cluster with config [\'kv,fts,n1ql,index\']: Error adding node: 172.23.109.179 to the cluster:172.23.109.172 - [&quot;Prepare join failed. This server does not have sufficient memory to support requested memory quota. Total quota is 3238MB (services: fts, index, kv, n1ql), maximum allowed quota for the node is 2950MB.&quot;]\n']</error>
	</testcase>
</testsuite>
